flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/JobTerminationMessageParameters.java

```diff
@@ -26,9 +26,10 @@
  *
  * <p>A job related REST handler always requires a {@link JobPathParameter}.
  */
-public class JobMessageParameters extends MessageParameters {
+public class JobTerminationMessageParameters extends MessageParameters {
```

Comment:
did you intentionally modify an existing class? Will this class not be used by several other handler sin the future as well?

Reply:
If need should arise, then we can add a `JobIdMessageParameters` class again. But right now it's not needed.